### ISSUE TITLE
Debug (-d) in pyjs broken

### ISSUE THREAD

**Issue by pyjsorg** [pyjs/original-pyjs#199]
<b>What steps will reproduce the problem?</b>
1. Checkout commit c6e72d3a474e25f9efd78049d105d2a87c2c0af4
2. Add bad code to Hello.py so as to create a runtime error
3. Compile with &quot;-d&quot; option.  Run in browser.

I expect to see a detailed traceback with line number.  Instead, I see the debug box, but it is blank.  Go back a commit, and all is well.

<b>If using PyJS, what compiler options did you use?</b>

-d

I'll see if I can figure out which commit on SF broke it.

_Original issue: http://code.google.com/p/pyjamas/issues/detail?id=520 (November 21, 2010 05:00:21)_

**Comment by pyjsorg** [comment]
_From gdwar...@gmail.com on November 21, 2010 05:27:41:_
I realized that the commit I mentioned above was a merge, so I dug further...

[gdw@ArchForever pyjamas]$ git bisect good
b9797af58da66f6197e5b009bbf294e3b07e2aa8 is the first bad commit
commit b9797af58da66f6197e5b009bbf294e3b07e2aa8
Author: Kees Bos cornelis.bos@gmail.com
Date:   Fri Nov 19 12:52:29 2010 +0100

```
Another fix for late defined variable/functions (discarded function call)
```

**Comment by pyjsorg** [comment]
_From cornelis...@gmail.com on November 21, 2010 07:32:06:_
Thanks. I'll look into this. Hope to find some time today, but will probably be tomorrow.

**Comment by pyjsorg** [comment]
_From cornelis...@gmail.com on November 22, 2010 07:45:13:_
Actually it was 0fd32a25edeae18246258df4ddc3a4bce72897de that was the first bad commit. Which was also mine :-)

Fixed this in 4820424200dddc25dd41bb5b491c1e589b9f2a10

Thanks Greg, for the report.
